script/Arcueid/Day 11/11_00_ARC11_2B.txt

```diff
@@ -125,7 +125,7 @@ I turn off the lamp.
 [sha:1be9bc406f89245f051ac91a5d51582021e3f3e8]{
 -- Page 156, Offset 29946.
 -- 時計は９時半。
-It's half-past nine.
+It's nine thirty.
```

Comment:
revert

Reply:
sorry I'm only following the format I saw before, for ex ARC7_3_2 `The mansion's lights are turned off at nine-thirty`, should I change all the similar ones?

Reply:
You needn't put so much emphasis on rigid consistency. Both are valid but "half-past nine" is a better independent thought. The other version in 7_3_2 is fine